pkg/util/namespaces/validation.go

```diff
@@ -168,6 +169,20 @@ func validateNamePlaceholderUsage(namePart, vclusterName, partTypeIdentifier, co
 	return nil
 }
 
+func validateHostMappingRules(hNS string, hIsPattern bool, configPathIdentifier, vclusterName, vclusterNamespace string) error {
+	// explicitly check against wildcard "catch all" mapping
+	if hIsPattern && hNS == WildcardChar {
```

Comment:
do you mind adding a test case for this? https://github.com/loft-sh/vcluster/blob/main/pkg/config/validation_test.go#L1163

Reply:
🫡  on it

Reply:
added 👍